core/controllers/editor.py

```diff
@@ -767,6 +767,28 @@ def get(self, exploration_id, escaped_state_name):
         })
 
 
+class FetchIssuesHandler(EditorHandler):
+    """Handler used for retrieving the list of unresolved issues in an
+    exploration.
+    """
+
+    @acl_decorators.can_view_exploration_stats
+    def get(self, exp_id):
+        """Handles GET requests."""
+        exp_issues = stats_services.get_exp_issues_by_id(exp_id)
```

Comment:
Please add error handling for invalid `exp_id` values.

Reply:
Done.